spring-boot-project/spring-boot-dependencies/build.gradle

```diff
@@ -1646,11 +1653,12 @@ bom {
 			]
 		}
 	}
-	library("Spring AMQP", "2.4.0-M1") {
+	library("Spring AMQP", "2.4.0-SNAPSHOT") {
```

Comment:
When's 2.4.0-M2 due? I can't see it in the release calendar at the moment.

Reply:
Aug 16 - scheduled.